.github/workflows/test.yml

```diff
@@ -3,7 +3,8 @@
 #
 
 name: FPC test
-on: 
+on:
+  workflow_dispatch: 
```

Comment:
@mbrandenburger  I think keeping workflow dispatch is a good addition to the CI, isn't it?

Reply:
Yes, it's nice to trigger the workflow manually.

Reply:
Thanks done